### PR TITLE
Cu 864dvj0b8 nav bar global julia

### PR DESCRIPTION
**Pull request by juhelena**
##### Description of changes

- [x] Why (Why did you make this pull request): Navbar global finalized

- [x] Brief description of the changes: Inclusion of images and change in menu hover
- [x] Possible breakouts (Possible places where things could break):

### CODE REVIEW

**Comment by vitormiura** [comment]
do not merge @abehidek @iguoliveira @viktormarinho, adding component class variants soon.

**Review by iguoliveira — COMMENTED** [review]
I'll start adding class variants to this component. In the next daily we can discuss about these changes. 😁

**Review by iguoliveira — APPROVED** [review]
Seems fine to merge!

File: apps/web-planner/src/pages/index.tsx [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import { useQuery } from "@tanstack/react-query";
 import { Link } from "fsr";
+import { useState } from "react";
 import { ButtonWebTemplate, Spinner } from "ui";
+import { NavBar } from "ui/base/NavBar";
```
Comment (viktormarinho) [inline apps/web-planner/src/pages/index.tsx:5]: This export should be coming from 'ui' too! dont know if you are not exporting it from index or just imported that way by accident

File: package.json [hunk 2]
```diff
@@ -34,5 +34,8 @@
   "engines": {
     "node": ">=14.0.0"
   },
-  "packageManager": "pnpm@7.14.2"
+  "packageManager": "pnpm@7.14.2",
+  "dependencies": {
+    "clsx": "^1.2.1"
```
Comment (viktormarinho) [inline package.json:39]: This should be removed and installed in the UI Package. 
after doing this, please do pnpm install again so the pnpm-lock.yml is updated

When you are installing packages, use pnpm add --filter <PACKAGE_NAME>

It wil prompt you and warn before installing global dependencies, you can only do it when 
passing the -w flag, so read the warning messages fully so you dont need to change dependency location again

**Review by viktormarinho — CHANGES_REQUESTED** [review]

**Review by vitormiura — APPROVED** [review]
just tested the component, and its working fine, just need to add clsx to ui pkg as @viktormarinho described above.

File: package.json (hunk 2 above)
Comment (iguoliveira) [reply package.json:39]: Thanks for warning! 😁

**Review by viktormarinho — APPROVED** [review]
Looks ok now!

**Pull request merged**